DevelopGitHub/GitHubExtension/GitHubExtension.LocalizationTool/MainWindow.xaml.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Windows;
+using System.Windows.Controls;
+
+using GitHubExtension.LocalizationTool.Translate;
+
+namespace GitHubExtension.LocalizationTool
+{
+    /// <summary>
+    ///     Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        private readonly Translator translator;
+
+        private readonly JsonHelper jsonHelper;
+
+        public MainWindow()
+        {
+            InitializeComponent();
+
+            TranslationData = new ObservableCollection<TranslationDataRow>();
+
+            translator = new Translator(this);
```

Comment:
I see smell. Class should't depend from Window because it's responsible for translation. All interaction with UI should be removed from class.